tests/phpunit/CRM/Event/Form/ParticipantTest.php

```diff
@@ -22,17 +22,11 @@ public function setUp(): void {
   }
 
   /**
-   * CHeck that all tests that have created payments have created them with the right financial entities.
+   * Should financials be checked after the test but before tear down.
    *
-   * Ideally this would be on CiviUnitTestCase but many classes would still fail. Also, it might
-   * be good if it only ran on tests that created at least one contribution.
-   *
-   * @throws \CRM_Core_Exception
+   * @var bool
    */
-  protected function assertPostConditions(): void {
```

Comment:
These asserts are now done by the parent if ``$isValidateFinancialsOnPostAssert = TRUE;`` is true